### ISSUE TITLE
No canvas output

### ISSUE THREAD

**Issue by acejacek** [martinRenou/ipycanvas#82]
I can't get any output. For example, this piece of code:
```
from ipycanvas import Canvas

canvas = Canvas(width=200, height=200)
canvas.fill_rect(25, 25, 100, 100)
canvas.clear_rect(45, 45, 60, 60)
canvas.stroke_rect(50, 50, 50, 50)
canvas
```
the only output I'm getting is:
```
Canvas(height=200, width=200)
```

I'm running than on MacOS, with brewed Python 3.7 and latest Jupyter. All installed according to the book, without Anaconda. No errors thrown, nothing to debug. All modules present, node v13, all are there waiting. Tested on multiple browsers too (Safari, Chrome, Firefox). 

I'm totally clueless. I'm new in the game, so maybe I'm doing a trivial error, but Google can't help me.

**Comment by martinRenou** [comment]
Thanks for opening a PR. 

- Are you using Jupyter Notebook or JupyterLab? If using JupyterLab, please don't forget to run `jupyter labextension install @jupyter-widgets/jupyterlab-manager ipycanvas`
- Did you install ipycanvas with conda? If yes, please re-install it doing `conda install -c conda-forge ipycanvas=0.4.3`. Something went wrong with the latest conda package, I am working on fixing it.

Otherwise, please provide the output of the following commands:
```
which jupyter
jupyter nbextension list
jupyter labextension list
```

**Comment by acejacek** [comment]
Hi,
I reinstalled everything from the scratch (making a deep puge of everything python-related), and now `ipycanvas ` works perfectly.
Thanks for all!

**Comment by martinRenou** [comment]
Happy to hear that it works! Don't hesitate to open other issues if you have any questions.

**Comment by MiguelCorti** [comment]
Hi! Didn't want to open a new issue as the problem I'm having seems very similar to this one, but I can't seem to solve it :(

I'm using Jupyter Notebook (no JupyterLab) without Anaconda. I'm using python 3.8.2 and installed ipycanvas 0.5.0 using pip. But I'm getting no output from canvas. I'm trying to run the following code from the the ipycanvas docs:

```python
from ipycanvas import Canvas

canvas = Canvas(width=200, height=200)

canvas.fill_rect(25, 25, 100, 100)
canvas.clear_rect(45, 45, 60, 60)
canvas.stroke_rect(50, 50, 50, 50)

canvas
```
The output is blank, but looking at my console (where jupyter server is running) I see the following output:
```
[W 10:57:08.758 NotebookApp] 404 GET /static/ipycanvas.js?v=20200903105459 (127.0.0.1) 1.01ms referer=http://localhost:8888/notebooks/trab_1.ipynb
```

And inspecting the browser console I see several errors as well:
![image](https://user-images.githubusercontent.com/5075086/92125062-ed66ac00-edd4-11ea-8586-dd66933b3e42.png)
I think it may be relevant to note that I'm using WSL with Ubuntu 20.04 as well.

Using ipycanvas on Jupyter Notebook through MyBinder gave me the exact same issue, but I was able to solve it running `!jupyter labextension install @jupyter-widgets/jupyterlab-manager ipycanvas`

As I don't have jupyter lab installed locally, I don't know what else to do.

Thanks! I can open a new issue if it seems appropiate.

**Comment by martinRenou** [comment]
Hey!

Can you run the following command in a terminal and show me the output:
```
jupyter nbextension list
```

**Comment by MiguelCorti** [comment]
Sure, sorry for taking long! I missed your comment...

Here is the output:
![image](https://user-images.githubusercontent.com/5075086/92317164-39337400-efd4-11ea-995d-58d3b849d96b.png)

**Comment by martinRenou** [comment]
Try running:

```bash
jupyter nbextension install --py --symlink --sys-prefix ipycanvas
jupyter nbextension enable --py --sys-prefix ipycanvas
```

This should not actually be needed. But it might fix your issue.

**Comment by MiguelCorti** [comment]
Thanksssssss, that worked! A friend of mine was having the same issue (using only notebook and pycanvas), will pass the knowledge on. Also, I had some problems with permissions when running both these commands, so I actually ran:
```
jupyter nbextension install --py --symlink --sys-prefix ipycanvas --user
jupyter nbextension enable --py ipycanvas
```

Added `--user` on the first command and removed `--sys-prefix` from the second one.

Everything seems to be working normally now. Thanks a lot!
